commands/class-run-all.php

```diff
@@ -14,18 +14,21 @@ class Today_Migration_All {
 		 * @when after_wp_load
 		 */
 		public function __invoke( $args ) {
-			$meta = new Today_Migration_Meta();
+			$content = new Today_Migration_Post_Content();
+			$content->__invoke( $args );
+
+			$meta    = new Today_Migration_Meta();
 			$meta->__invoke( $args );
 
-			$feat = new Today_Migration_Featured_Image();
+			$feat    = new Today_Migration_Featured_Image();
```

Comment:
We can probably rename this variable `$feature` now.. I only made it `$feat` so it would line up nicely 😄

Reply:
Updated :)